core/src/main/scala/kafka/tools/ConsoleProducer.scala

```diff
@@ -71,7 +71,10 @@ object ConsoleProducer {
   }
 
   def getReaderProps(config: ProducerConfig): Properties = {
-    val props = new Properties
+    val props =
+      if (config.options.has(config.readerConfigOpt))
+        Utils.loadProps(config.options.valueOf(config.readerConfigOpt))
+      else new Properties
```

Comment:
nit: Could we format it as follow?

```
val props = if (config.options.has(config.readerConfigOpt))
  Utils.loadProps(config.options.valueOf(config.readerConfigOpt))
else
  new Properties
```

Reply:
It was written the same way as in next method (https://github.com/apache/kafka/pull/12175/files#diff-3b7393aaaffd466b3bfc53c8d95109a206442160b878bbdaa0b0332257afaebaR84-R87)
Do you want me rewrite it as well?